lib/ansible/module_utils/network/ios/config/vlans/vlans.py

```diff
@@ -146,16 +146,22 @@ def _state_overridden(self, want, have, state):
         """
         commands = []
 
+        want_local = want
```

Comment:
why aren't you just using want? I haven't seen anything that would prevent you from it

Reply:
@danielmellado wanted to work on the local copy of want, but yes I can directly work on want dict itself, I'll update the change.

Reply:
ack, thanks!